src/polkit/polkitauthority.c

```diff
@@ -287,6 +298,21 @@ polkit_authority_class_init (PolkitAuthorityClass *klass)
                                           g_cclosure_marshal_VOID__VOID,
                                           G_TYPE_NONE,
                                           0);
+  /**
+   * PolkitAuthority::sessions-changed:
+   * @authority: A #PolkitAuthority.
+   *
+   * Emitted when sessions change
+   */
+  signals[SESSIONS_CHANGED_SIGNAL] = g_signal_new ("sessions-changed",
+                                            POLKIT_TYPE_AUTHORITY,
```

Comment:
This (and the lines below) should be probably all indented under `"sessions-changed"`.

Reply:
True, fixed.